update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
<pre>
#!/usr/bin/env bash

# Authors:
#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]

# Description:
#   An update bash script for engelsystem

# Add upstream 
git remote add upstream http://www.github.com/DishantK1807/engelsystem.git

# Pull the updates
git pull upstream auto-update
</pre>

This is the script that I used for testing on my local server. It includes pulling from my repository, pull the auto-update branch. I changed the repo in the script while pushing the code to fossasia.